xwayland/xwm.c

```diff
@@ -270,6 +278,13 @@ static void xwm_send_focus_window(struct wlr_xwm *xwm,
 	values[0] = XCB_STACK_MODE_ABOVE;
 	xcb_configure_window(xwm->xcb_conn, xsurface->window_id,
 		XCB_CONFIG_WINDOW_STACK_MODE, values);
+
+	xsurface_set_net_wm_state(xsurface);
+unfocus:
```

Comment:
Maybe move the `if` block at the top of the function, so that we don't need to have `goto`s?